docs/_posts/2015-10-07-react-v0.14.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "React v0.14"
+author: spicyj
+---
+
+We’re happy to announce the release of React 0.14 today! This release has a few major changes, primarily designed to simplify the code you write every day and to better support environments like React Native.
+
+If you tried the release candidate, thank you – your support is invaluable and we've fixed a few bugs that you reported.
+
+As with all of our releases, we consider this version to be stable enough to use in production and recommend that you upgrade in order to take advantage of our latest improvements.
+
+## Upgrade Guide
+
+Like always, we have a few breaking changes in this release. We know changes can be painful (the Facebook codebase has over 15,000 React components), so we always try to make changes gradually in order to minimize the pain.
+
+If your code is free of warnings when running under React 0.13, upgrading should be easy. We have two new small breaking changes that didn't give a warning in 0.13 (see below). Every new change in 0.14, including the major changes below, is introduced with a runtime warning and will work as before until 0.15, so you don't have to worry about your app breaking with this upgrade.
+
+See the changelog below for more details.
+
+## Installation
+
+We recommend using React from `npm` and using a tool like browserify or webpack to build your code into a single bundle. To install the two packages:
```

Comment:
I don't think we should downplay the prebuilt browser builds.  I think they're as important (perhaps more important) than the NPM modules.  They should certainly be on the same footing.